worldmap/config/journeymap.core.config

```diff
@@ -12,11 +12,12 @@
   "announceMod": "true",
   "checkUpdates": "true",
   "recordCacheStats": "false",
-  "browserPoll": "2000",
   "themeName": "OceanMonument",
   "caveIgnoreGlass": "true",
   "mapBathymetry": "false",
+  "mapWaterBiomeColors": "false",
   "mapTopography": "true",
+  "mapBiome": "true",
```

Comment:
Do wynncraft have proper biomes? Or does this risk giving an incorrect tint to areas of different biomes?